Sources/NIO/SelectorGeneric.swift

```diff
@@ -121,18 +121,15 @@ protocol _SelectorBackendProtocol {
 /* this is deliberately not thread-safe, only the wakeup() function may be called unprotectedly */
 internal class Selector<R: Registration>  {
     var lifecycleState: SelectorLifecycleState
-
-    // temporary workaround to stop us delivering outdated events; read in `whenReady`, set in `deregister`
-    var deregistrationsHappened: Bool = false
     var registrations = [Int: R]()
+    var sequenceIdentifier : RegistrationSequenceIdentifier = 1
```

Comment:
Pedantic nit, but NIO style:

```suggestion
    var sequenceIdentifier: RegistrationSequenceIdentifier = 1
```